### ISSUE TITLE
range pin on scikit-learn is not permissive enough to use modern versions of skl

### ISSUE THREAD

**Issue by kwlzn** [tensorflow/data-validation#93]
when we try to compose tfdv into a python resolve along with the latest scikit-learn, we hit a resolver failure due to a transitive pin on `scikit-learn<0.21,>=0.18`:

```
Exception message: Could not satisfy all requirements for scikit-learn==0.21.3; python_version >= "3":
    scikit-learn==0.21.3; python_version >= "3", scikit-learn<0.21,>=0.18(from: tensorflow_data_validation==0.14.1)
```

would it be possible to loosen this constraint?

**Comment by rmothukuru** [comment]
Please provide details about what platform you are using (operating system, architecture). 

Make sure you also include the exact command if possible to produce the output included in your test case. If you are unclear what to include see the issue template displayed in [the Github new issue template](https://github.com/tensorflow/tensorflow/issues/new/choose).

We ask for this in the issue submission template, because it is really difficult to help without that information. Thanks!

**Comment by paulgc** [comment]
@kwlzn This issue should be fixed with TFDV 0.15 which was released today.

**Comment by kwlzn** [comment]
@paulgc awesome, thanks a ton for the quick turnaround!

confirmed this fixes the above issue, but we're now constrained by:

```
Exception message: Could not satisfy all requirements for tensorflow_data_validation==0.15.0:
    tensorflow_data_validation==0.15.0, tensorflow-data-validation<0.15,>=0.14.1(from: tfx==0.15.0dev+twtr0)
```

so will await a `tfx` release that permits this new version to unblock.
